packages/react-components/src/components/table-output-component.tsx

```diff
@@ -466,29 +463,21 @@ export class TableOutputComponent extends AbstractOutputComponent<TableOutputPro
         }
     }
 
-    private handleRowSelectionChange(load?: any | undefined) {
+    private selectAndNavigateToState(load?: any | undefined) {
+        const startTimestamp = this.startTimestamp;
+        const endTimestamp = this.endTimestamp;
         if (this.timestampCol) {
-            const startTimestamp = String(this.startTimestamp);
-            const endTimestamp = String(this.endTimestamp);
-            const payload = { startTimestamp, endTimestamp, load };
-            this.prevStartTimestamp = BigInt(startTimestamp);
-            this.eventSignal = true;
-            this.handleSelectionRangeUpdate(payload);
-            signalManager().fireSelectionChangedSignal(payload);
-        }
-    }
-
-    private handleSelectionRangeUpdate(payload: { [key: string]: string; }) {
-        const offset = this.props.viewRange.getOffset() || BigInt(0);
-        const startTimestamp = payload['startTimestamp'];
-        const endTimestamp = payload['endTimestamp'];
-        if (startTimestamp !== undefined && endTimestamp !== undefined) {
-            const selectionRangeStart = BigInt(startTimestamp) - offset;
-            const selectionRangeEnd = BigInt(endTimestamp) - offset;
-            this.props.unitController.selectionRange = {
-                start: selectionRangeStart,
-                end: selectionRangeEnd
-            };
```

Comment:
**Refactor:** This was an existing implementation of changing the selection range from signals.  It triggered when a user clicked on an event in the Events Table, and TimeGraphOutputComponents found that event and ensured it was visible on the screen.  It also changed the selection range.